client/lib/analytics/index.js

```diff
@@ -125,6 +125,11 @@ var analytics = {
 		}
 	},
 
+	recordEvent: function( tracksEventName, tracksEventProperties, category, action, label, value ) {
```

Comment:
@apeatling - do you think we could standardize the naming between Google Analytics and Tracks events?